tests/lib/pages/base_page.rb

```diff
@@ -1,8 +1,19 @@
+require 'json'
```

Comment:
if we leave this line here than our test project will require this lib each time we require this file
the better option would be to move it to env.rb